mtags/src/main/scala-2/scala/meta/internal/pc/completions/ScalaCliCompletions.scala

```diff
@@ -34,6 +34,16 @@ trait ScalaCliCompletions {
       dependency: String
   ) extends CompletionPosition {
 
+    override def compare(o1: Member, o2: Member): Int =
+      (o1, o2) match {
+        case (c1: TextEditMember, c2: TextEditMember) =>
```

Comment:
Would it make sense to create DependencyMember and use SemVer class we have to compare?

Reply:
What abou this? Did you manage to get this working?

Reply:
I've added `DependencyMember` and reused it also in completions for `sbt` and `mill` (I've completely forgotten about them before).